spark/dl/src/main/scala/com/intel/analytics/bigdl/example/treeLSTMSentiment/fetch_and_preprocess.py

```diff
@@ -0,0 +1,392 @@
+"""
```

Comment:
This won't be packaged to dist, I think. So how does user run this example with a prebuild BigDL package.